tables/he-common-vowels-ihbc.uti

```diff
@@ -0,0 +1,480 @@
+always בֶּ 12-15  # bet  with dagesh and segol
+always בַּ 12-14  # bet  with dagesh and patah
+always בָּ 12-126  # bet dagesh qamats
+always גְּ 5-1245-3  # gimel with dagesh and sheva
+always גֱּ 5-1245-26  # Gimel dagesh  hataf segol
+always גֲּ 5-1245-25  # Gimel dagesh  hataf patah
+always גֳּ 5-145-345  # Gimel dagesh  hataf qamats
+always גִּ 5-1245-24  # gimel with dagesh and hiriq
+always גֵּ 5-1245-34  # gimel with dagesh and tsere
+always גֶּ 5-1245-15  # gimel with dagesh and segol
+always גַּ 5-1245-14  # gimel with dagesh and patah
+always גָּ 5-1245-126  # Gimel dagesh  qamats
+always גְּ 5-1245-3  # gimel with dagesh and sheva
+always גֱּ 5-1245-26  # Gimel dagesh  hataf segol
+always גֲּ 5-1245-25  # Gimel dagesh  hataf patah
+always גֳּ 5-145-345  # Gimel dagesh  hataf qamats
+always גִּ 5-1245-24  # gimel with dagesh and hiriq
+always גֵּ 5-1245-34  # gimel with dagesh and tsere
+always גֶּ 5-1245-15  # gimel with dagesh and segol
+always גַּ 5-1245-14  # gimel with dagesh and patah
+always גָּ 5-1245-126  # Gimel dagesh  qamats
+always דְּ 5-145-3  # dalet with dagesh sheva
+always דֱּ 5-145-26  # dalet with dagesh and hataf segol
+always דֲּ 5-145-25  # dalet with dagesh and hataf patah
+always דֳּ 5-145-345  # dalet with dagesh and hataf qamats
+always דִּ 5-145-24  # dalet  with dagesh and hiriq
+always דֵּ 5-145-34  # dalet with dagesh and tsere
+always דֶּ 5-145-15  # dalet with dagesh and segol
+always דַּ 5-145-14  # dalet with dagesh and patah
+always דָּ 5-145-126  # dalet with dagesh and qamats
+always דְּ 5-145-3  # dalet with dagesh sheva
+always דֱּ 5-145-26  # dalet with dagesh and hataf segol
+always דֲּ 5-145-25  # dalet with dagesh and hataf patah
+always דֳּ 5-145-345  # dalet with dagesh and hataf qamats
+always דִּ 5-145-24  # dalet  with dagesh and hiriq
+always דֵּ 5-145-34  # dalet with dagesh and tsere
+always דֶּ 5-145-15  # dalet with dagesh and segol
+always דַּ 5-145-14  # dalet with dagesh and patah
+always דָּ 5-145-126  # dalet with dagesh and qamats
+always הְּ 5-125-3  # hey with dagesh and sheva
+always הֱּ 5-125-26  # Hey dagesh  hataf segol
+always הֲּ 5-125-25  # Hey dagesh  hataf patah
+always הֳּ 5-125-345  # Hey dagesh  hataf qamats
+always הִּ 5-125-24  # heywith dagesh and hiriq
+always הֵּ 5-125-34  # Hey with dagesh and tsere
+always הֶּ 5-125-15  # Hey with dagesh and segol
+always הַּ 5-125-14  # Hey with dagesh and patah
+always הָּ 5-125-126  # Hey dagesh  qamats
+always הְּ 5-125-3  # hey with dagesh and sheva
+always הֱּ 5-125-26  # Hey dagesh  hataf segol
+always הֲּ 5-125-25  # Hey dagesh  hataf patah
+always הֳּ 5-125-345  # Hey dagesh  hataf qamats
+always הִּ 5-125-24  # hey with dagesh and hiriq
+always הֵּ 5-125-34  # Hey with dagesh and tsere
+always הֶּ 5-125-15  # Hey with dagesh and segol
+always הַּ 5-125-14  # Hey with dagesh and patah
+always הָּ 5-125-126  # Hey dagesh  qamats
+always וֹ 135  # holam-vav
+always וּ 346  # shuruq
+always ּו 346  # shuruq
```

Comment:
This line is really wrong and should be removed. The dagesh should not come before the vav.
```suggestion
```